src/main/java/com/redhat/devtools/intellij/common/utils/ConfigWatcher.java

```diff
@@ -38,30 +38,22 @@ public class ConfigWatcher implements Runnable {
     private WatchService service;
 
     public interface Listener {
-        void onUpdate(ConfigWatcher source, Config config);
+        void onUpdate(Config updatedConfig);
     }
 
-    public ConfigWatcher(String config, Listener listener) {
-        this(Paths.get(config), listener);
+    public ConfigWatcher(Listener listener) {
+        this(Config.getKubeconfigFilenames().stream().map(Paths::get).toList(), listener, new HighSensitivityRegistrar());
```

Comment:
`ConfigWatcher` does not need to get the path to the config file. It is using the Client config which merged all the config files that were merged when it (auto-) configured